src/main/java/com/dev/cinema/dao/CinemaHallDao.java

```diff
@@ -2,9 +2,12 @@
 
 import com.dev.cinema.model.CinemaHall;
 import java.util.List;
+import java.util.Optional;
 
 public interface CinemaHallDao {
     CinemaHall add(CinemaHall cinemaHall);
 
     List<CinemaHall> getAll();
+
+    Optional<CinemaHall> get(Long cinemaHallId);
```

Comment:
maybe, it`s better to rename it to getById, so we could understand what exactly is going on